man/ocamldoc.m

```diff
@@ -236,6 +236,11 @@ is a list of complete module names are separated by commas (,),
 .BR \-latex , \ \-texi ,\ or \ \-dot
 options.
 .TP
+.BI \-open \ module
+Opens
+.IR module
```

Comment:
My understanding of the [groff manual](https://www.gnu.org/software/groff/manual/html_node/Man-font-macros.html) is that this should be `.I` rather than `.IR`, the latter being a two-argument macro -- we used it below with the second argument being a *dot*, I think.